src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -283,21 +285,29 @@ public static void AssertValidVariableValue(ISchema schema, IGraphType type, obj
         /// <summary>
         /// Coerces a variable value to a compatible .NET type for the variable's graph type.
         /// </summary>
-        public static object CoerceValue(ISchema schema, IGraphType type, IValue input, Variables variables = null)
+        public static object CoerceValue(ISchema schema, IGraphType type, IValue input, Variables variables = null, object fieldDefault = null)
```

Comment:
CoerceValue now accepts an additional argument specifying the default field value if none was specified.  Previously, it would return null if none was specified.  As such, the semantics of `CoerceValue` have not changed with this PR unless you supply new additional argument.